### PR TITLE
update getCredentials method

### PR DESCRIPTION
**Pull request by jinyangyang222**
fiexd #447

### CODE REVIEW

**Review by swaroopar — CHANGES_REQUESTED** [review]
comments added.

File: modules/orchestrator/src/main/java/org/eclipse/xpanse/orchestrator/credential/CredentialCenter.java [hunk 1]
```diff
@@ -319,4 +326,35 @@ private boolean isAnyMandatoryCredentialVariableMissing(
                         && Objects.isNull(credentialVariable.getValue()));
     }
 
+    private void checkNullParamAndFillValueFromEnv(AbstractCredentialInfo abstractCredentialInfo) {
+        if (abstractCredentialInfo.getType().equals(CredentialType.VARIABLES)) {
+            CredentialVariables credentialVariables =
+                    (CredentialVariables) abstractCredentialInfo;
+            for (CredentialVariable variable : credentialVariables.getVariables()) {
+                if (Objects.isNull(variable.getValue())
+                        || Objects.equals("null", variable.getValue())) {
```
[the remark this answers is not in the document]
Comment (jinyangyang222) [reply modules/orchestrator/src/main/java/org/eclipse/xpanse/orchestrator/credential/CredentialCenter.java]: fixed.

File: modules/orchestrator/src/main/java/org/eclipse/xpanse/orchestrator/credential/CredentialCenter.java [hunk 2]
```diff
@@ -319,4 +326,35 @@ private boolean isAnyMandatoryCredentialVariableMissing(
...
+                        || Objects.equals(variable.getValue(), "null")) {
+                    variable.setValue(System.getenv(variable.getName()));
+                }
+            }
+        }
+    }
+
+    private void addCredentialInfoFromEnv(Csp csp,
+                                          List<AbstractCredentialInfo> abstractCredentialInfos) {
+        AbstractCredentialInfo credentialInfoFromEnv = findCredentialInfoFromEnv(csp);
```
Comment (swaroopar) [inline modules/orchestrator/src/main/java/org/eclipse/xpanse/orchestrator/credential/CredentialCenter.java:344]: findCredentialInfoFromEnv - can return null. 
So we need a check here to add only non-null values here.

**Review by swaroopar — CHANGES_REQUESTED** [review]
@jinyangyang222 small comment added.

File: modules/orchestrator/src/main/java/org/eclipse/xpanse/orchestrator/credential/CredentialCenter.java (hunk 2 above)
Comment (jinyangyang222) [reply modules/orchestrator/src/main/java/org/eclipse/xpanse/orchestrator/credential/CredentialCenter.java:344]: ok ok, I made an oversight, it's fixed.

**Review by swaroopar — APPROVED** [review]
@jinyangyang222 thank you. Looks good.

**Pull request merged**